src/components/timeline/_macro.njk

```diff
@@ -3,8 +3,8 @@
 
     <div class="ons-timeline{{ ' ' + params.classes if params.classes else '' }}">
         {% set titleTag = params.headingLevel | default(2) %}
-        {% set openingTag = "<h" + titleTag %}
-        {% set closingTag = "</h" + titleTag + ">" %}
+        {% set openingTag = "<h" ~ titleTag %}
+        {% set closingTag = "</h" ~ titleTag + ">" %}
```

Comment:
Would this also need to be `</h" ~ titleTag ~ ">"`?

Reply:
updated

> Would this also need to be `</h" ~ titleTag ~ ">"`?

update